tests/integration/security/egress_gateway_origination/testdata/destination-rule-no-tls-origination.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: DestinationRule
+metadata:
+  name: originate-tls-for-edition-cnn-com
+spec:
+  host: edition.cnn.com
+  trafficPolicy:
+    loadBalancer:
+      simple: ROUND_ROBIN
```

Comment:
nit: often tests are used as examples, so it may make sense to keep the config minimal. Otherwise its hard to tell if ROUND_ROBIN is here because its required or because you copy and pasted from somewhere that has it set